slides/week3/week3.org

```diff
@@ -0,0 +1,54 @@
+#+TITLE: Week 3
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* TODO Needs speaker notes
+
+* What are we doing today?
+- Data structures and algorithms in C++
+- Brancing and merging in Git
+
+* What's a constructor?
+- A special type of function that instantiates a class
+- Basically, it sets up an object and returns it
+
+* What's a container?
+- An object you can use to store other objects
+- They can be ordered, associative, both, or neither
+  - *Ordered:* The contents are in a specific order
+  - *Associative:* Two objects are associated with each other within the container
+- TODO am I missing any important properties here? I want to say "uniqueness" or something...
```

Comment:
Here's an attempt 😢 
Containers hold pieces of data
Index: You need a way to index into any container, be it an assigned number, the data itself, or another value you provide. This is called the 'Key' or 'Index' of a piece of data.
Unique Keys: No two data entries can have the same index. If they could, how would C++ decide which piece of data to give you?